src/extension.ts

```diff
@@ -175,6 +178,31 @@ async function detectTknCli(): Promise<void> {
 
   if (tknPath) {
     setCommandContext(CommandContext.TknCli, true);
+    sendVersionToTelemetry('tkn.version', tknPath);
+  }
+  sendVersionToTelemetry('kubectl.version', 'kubectl -o json');
+}
+
+async function sendVersionToTelemetry(commandId: string, cmd: string): Promise<void> {
+  const telemetryProps: TelemetryProperties = {
+    identifier: commandId,
+  };
+  const result = await cli.execute(createCliCommand(`${cmd} version`));
+  if (result.error) {
+    telemetryProps.error = result.error.toString();
+    sendTelemetry('command', telemetryProps);
+  } else {
+    if (commandId === 'kubectl.version') {
```

Comment:
I think that we need same for tkn versions, I mean that we need to store each of tkn cli, tekton version and triggers versions as separate values, that will help us to analyze used versions more easy.

Reply:
when we store `tkn version` it looks like `Client version: 0.15.0 Pipeline version: v0.16.3 Triggers version: v0.8.1` which is similar to what I am doing with kubectl.

Reply:
Yes, that what I was thinking, if we store all tekton related versions in one row. The in case if we want to measure how much of our users use some specific  `Pipeline version` it may be a bit difficult.

Reply:
You mean we should send the tkn related version separately?

Reply:
I'm not 100% sure, I think you can ask Fred how to do it better.

Reply:
@fbricon What's your opinion on sending data for `tkn version` it contains `Client, Pipeline, and trigger version` for example see in image. Should we send them separately?

<img width="373" alt="Screenshot 2021-02-10 at 1 40 36 PM" src="https://user-images.githubusercontent.com/9924513/107482454-b7a8c900-6ba5-11eb-949a-971180bac433.png">

Reply:
yes, that's 3 properties